showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITVersionHeaders.java

```diff
@@ -323,4 +331,26 @@ void testHttpJsonCompliance_userApiVersionSetSuccess() throws IOException {
       assertThat(headerValue).isEqualTo(CUSTOM_API_VERSION);
     }
   }
+
+  @Test
+  void testGrpc_matchesClientLibVersion() {
+    Pattern defautlGrpcHeaderPattern =
+        Pattern.compile("gl-java/.+ gapic/.+--protobuf-.+ gax/.+ grpc/.+ protobuf/.*");
+    grpcClient.echo(EchoRequest.newBuilder().build());
+    String headerValue = grpcInterceptor.metadata.get(API_CLIENT_HEADER_KEY);
+
+    assertTrue(defautlGrpcHeaderPattern.matcher(headerValue).matches());
+  }
+
+  @Test
+  void testHttpJson_matchesClientLibVersion() {
+    Pattern defautlHttpHeaderPattern =
+        Pattern.compile("gl-java/.+ gapic/.+--protobuf-.+ gax/.+ rest/ protobuf/.*");
+    httpJsonClient.echo(EchoRequest.newBuilder().build());
+    ArrayList headerValues =
+        (ArrayList) httpJsonInterceptor.metadata.getHeaders().get(HTTP_CLIENT_API_HEADER_KEY);
```

Comment:
Rather than using raw-types, use `ArrayList<?>` if you don't know the type. In this case, we're casting to String in the next line so we might as well cast to `ArrayList<String>`

Reply:
done